packages/flutter_driver/test/src/real_tests/extension_test.dart

```diff
@@ -984,6 +988,95 @@ void main() {
     });
   });
 
+  group('extension commands', () {
+    final VoidCallback mockCallback = MockCallback();
```

Comment:
Please avoid using mockito for something like this, and just have an instance variable somewhere that tracks when this gets called or how many times it gets called.

Reply:
Fine - I've removed mockito in favour of `VoidCallback`.